packages/trader/src/AppV2/Components/CurrentSpot/current-spot.scss

```diff
@@ -0,0 +1,107 @@
+.trade__current-spot {
+    position: relative;
+    border-radius: var(--core-borderRadius-400);
+    background-color: var(--core-color-solid-slate-50);
+    height: var(--core-size-2400);
+    overflow: hidden;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    .current-spot {
+        display: flex;
+        gap: var(--core-spacing-100);
+
+        p {
```

Comment:
We are trying to rewrite quill style here by adding styles for tag?

Reply:
lol bruh